rca/api_content/templates/api_content/api_content_admin.html

```diff
@@ -0,0 +1,32 @@
+{% extends "wagtailadmin/base.html" %}
+{% load wagtailadmin_tags static i18n %}
+{% block titletag %}{% trans "Dashboard" %}{% endblock %}
+{% block bodyclass %}homepage{% endblock %}
+
+
+{% block content %}
+    <script>
+    function clear_cache() {
+        $.get('/admin/api-cache/clear/', function (data) {
+            $("#cache_clear_button").removeClass("button-longrunning-active")
+            $("#cache_clear_button").prop("disabled", false);
+            $('.content').prepend('<div class="messages"><ul><li class="success">Success! API feed cache has been cleared</li></ul></div>')
```

Comment:
I think having a hidden div is better than dynamically generating HTML code in JS.

```js
document.getElementById('success-message').style.display = '';
```